crates/re_data_source/src/data_loader/loader_external.rs

```diff
@@ -78,14 +83,16 @@ impl crate::DataLoader for ExternalLoader {
 
         re_tracing::profile_function!(filepath.display().to_string());
 
+        let (tx_feedback, rx_feedback) = std::sync::mpsc::channel();
```

Comment:
The name and use of this is quite unclear until the reader has read the entire function.

It could be improved by naming the message, i.e. replacing `()` with `struct CompatibleLoaderFound{}` or similar